applications/plugins/LMConstraint/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.12)
+project(LMConstraint VERSION 1.0 LANGUAGES CXX)
+
+find_package(SofaFramework REQUIRED)
+find_package(SofaBase REQUIRED)
+find_package(SofaCommon REQUIRED) 
+find_package(SofaConstraint REQUIRED) 
+find_package(SofaDistanceGrid QUIET) 
```

Comment:
If you want to use the variable LMCONSTRAINT_HAVE_SOFADISTANCEGRID (e.g in your cmake.in file or config.h.in), you need to use sofa_find_package instead()